site/en/about/milvus_adopters.md

```diff
@@ -8,31 +8,31 @@ Milvus is trusted by over 1,000 organizations worldwide and has applications in
 
 | **Company**                                                  | **Industry**           | **User Story**                                               |
 | ------------------------------------------------------------ | ---------------------- | ------------------------------------------------------------ |
+| [ambeRoad](https://amberoad.de/?lang=en) |  Software| |
 | [Beike](https://investors.ke.com/about-us/default.aspx)                                                | Real estate           | [Making With Milvus: AI-Infused Proptech for Personalized Real Estate Search](https://zilliz.com/blog/Making-With-Milvus-AI-Infused-Proptech-for-Personalized-Real-Estate-Search)                                              |
 | [DXY.cn](https://en.wikipedia.org/wiki/DXY.cn)                                                  | Medical           |                                             |
 |[Deepset.ai](https://deepset.ai/)                                                    | Software         | [Semantic Search with Milvus, Knowledge Graph QA, Web Crawlers and more!](https://medium.com/deepset-ai/semantic-search-with-milvus-knowledge-graph-qa-web-crawlers-and-more-837451eae9fa)                                             |
```

Comment:
Please put Deepset.ai in front of DXY.cn

Reply:
Okay. I am on it !